### PR TITLE
Updated FAQ section with date/time/location

### PR DESCRIPTION
**Pull request by jsi19**
Also created variables for date/time/location and concatenated into text.

### CODE REVIEW

**Review by SubramaniamSatyen — CHANGES_REQUESTED** [review]
On line 152, should use the hothNum variable. Looks good otherwise

File: src/components/HomePage/FAQSection.js [hunk 1]
```diff
@@ -110,6 +110,10 @@ AccordionQA.propTypes = {
 };
 
 function FAQSection() {
+	const hothNum = 'X';
+	const appDeadline = 'Friday, February 24, 2023';
+	const hothDate = 'Sunday, March 5, 2023';
```
Comment (katelynsyu) [inline src/components/HomePage/FAQSection.js:115]: I really like this idea to have everything in a constant and just call the constant. Turns out we have a file already with constants in it, called constants.js under components. if you could move these constants to that file and update the constants there, that would be amazing

**Review by jsi19 — COMMENTED** [review]
Fixed

**Review by katelynsyu — CHANGES_REQUESTED** [review]

**Review by katelynsyu — CHANGES_REQUESTED** [review]
On line 177, there's an extra exclamation mark that shouldn't be there

File: src/components/HomePage/FAQSection.js (hunk 1 above)
Comment (jsi19) [reply src/components/HomePage/FAQSection.js:115]: Will do when available if not assigned to other

**Review by jsi19 — COMMENTED** [review]
Fixed

**Review by SubramaniamSatyen — APPROVED** [review]
If we're holding off on moving the variables to a shared place across the site, looks good!

**Review by katelynsyu — APPROVED** [review]
lgtm

**Pull request merged**